### PR TITLE
ignitions: disable kernel.hung_task_panic

### PR DESCRIPTION
**Pull request by ymmt2005**
Programs that use Ceph RBD block device can hang longer than
120 seconds, and let the kernel panic because of this setting.

Disable it to prevent panics at least for now.

### CODE REVIEW

**Review by toshipp — APPROVED** [review]
LGTM

**Comment by ymmt2005** [comment]
@toshipp Test failed because `artifacts.go` is stale.  Please take another look.

**Review by ysksuzuki — DISMISSED** [review]

File: artifacts.go [hunk 1]
```diff
@@ -5,16 +5,15 @@ package neco
 
 var CurrentArtifacts = ArtifactSet{
 	Images: []ContainerImage{
-		{Name: "coil", Repository: "ghcr.io/cybozu-go/coil", Tag: "2.0.6", Private: false},
+		{Name: "coil", Repository: "ghcr.io/cybozu-go/coil", Tag: "2.0.7", Private: false},
 		{Name: "bird", Repository: "quay.io/cybozu/bird", Tag: "2.0.7.5", Private: false},
 		{Name: "chrony", Repository: "quay.io/cybozu/chrony", Tag: "4.0.0.1", Private: false},
 		{Name: "etcd", Repository: "quay.io/cybozu/etcd", Tag: "3.3.25.4", Private: false},
-		{Name: "promtail", Repository: "quay.io/cybozu/promtail", Tag: "2.1.0.1", Private: false},
```
Comment (ysksuzuki) [inline artifacts.go:12]: @ymmt2005 This should not be removed.  Your `generate-artifacts` is a bit old perhaps.

Comment (ymmt2005) [reply artifacts.go]: oops.. fixed.

**Review by ysksuzuki — APPROVED** [review]
LGTM

**Pull request merged**